ddtrace/_trace/processor/__init__.py

```diff
@@ -283,9 +284,11 @@ def __init__(
                 agent_url=agent_config.trace_agent_url,
                 dogstatsd=get_dogstatsd_client(agent_config.dogstatsd_url),
                 sync_mode=SpanAggregator._use_sync_mode(),
-                headers={"Datadog-Client-Computed-Stats": "yes"}
-                if (config._trace_compute_stats or asm_config._apm_opt_out)
-                else {},
+                headers=(
```

Comment:
why this change?

Reply:
I removed the change so it's not there in the latest commit anymore.